packages/jest-cli/src/testResultHelpers.js

```diff
@@ -85,6 +85,12 @@ export const addResult = (
   aggregatedResults: AggregatedResult,
   testResult: TestResult,
 ): void => {
+  // `todos` are new as of Jest 24, and not all runners return it.
+  // Set it to `0` to avoid `NaN`
+  if (!testResult.numTodoTests) {
+    testResult.numTodoTests = 0;
```

Comment:
isn't this something that runners should add themselves for Jest 24 compat, like we did in `create-jest-runner`?

Reply:
sure, but we shouldn't require runners to do so. The fact they have to return a huge results object at all is pretty sad. The real fix IMO would be for runners to return something minimal, and we do a `const result = {...defaultResultObject, ...returnValueFromRunner}` (something more clever for deep merge, but essentially this). Should be enough to return e.g. `{pass: true, start: timestamp, end: timestamp}` or something like that

Reply:
Couldn't agree more